.github/workflows/push_image_pr.yml

```diff
@@ -7,6 +7,7 @@ env:
   WF_REGISTRY_USER: netobserv+github_ci
   WF_REGISTRY: quay.io/netobserv
   WF_IMAGE: netobserv-ebpf-agent
+  WF_BC_IMAGE: ebpf-bytecode
```

Comment:
Could we have consistency in repo names here ?
I saw it was `ebpf-bytecode` instead of `netobserv-ebpf-bytecode` in the initial script :thinking:

Reply:
the image name needs to be in sync with the operator repo
https://github.com/netobserv/network-observability-operator/blob/main/main.go#L113

Reply:
@jpinsonneau yes it's `ebpf-bytecode`, is there smthg wrong with it? the name was briefly discussed here https://github.com/netobserv/netobserv-ebpf-agent/pull/443#discussion_r1877929570

Reply:
Just for consistency @jotak ; but in the end none of our repo are consistents... sometimes we prefix network-observability-, sometimes netobserv- and else we don't prefix. So let's keep it as is now.

Reply:
let's add another standard on top of all the existing ones 😂

Reply:
(I'm joking, but I'm ok to rename the repo)